File: utilsj/src/main/java/com/ung8023/utilsj/DrawableUtils.java
package com.ung8023.utilsj;

import android.content.Context;
import android.content.res.Resources;
import android.graphics.drawable.GradientDrawable;
import android.support.annotation.DrawableRes;
import android.support.annotation.NonNull;

public class DrawableUtils {
    public static GradientDrawable getShape(@NonNull Context context, @DrawableRes int drawableRes) {
        return (GradientDrawable) context.getResources().getDrawable(drawableRes);
    }

    public static GradientDrawable getShape(@NonNull Context context, @DrawableRes int drawableRes, Resources.Theme theme) {
        return (GradientDrawable) context.getResources().getDrawable(drawableRes, theme);
    }
}

File: utilskt/src/main/kotlin/com/ung8023/utilkt/DrawableUtils.kt
package com.ung8023.utilkt

import android.content.Context
import android.content.res.Resources
import android.graphics.drawable.GradientDrawable
import android.support.annotation.DrawableRes

object DrawableUtils {
    fun getShape(context: Context, @DrawableRes drawableRes: Int): GradientDrawable? {
        return context.resources.getDrawable(drawableRes) as GradientDrawable?
    }

    fun getShape(context: Context, @DrawableRes drawableRes: Int, theme: Resources.Theme): GradientDrawable? {
        return context.resources.getDrawable(drawableRes, theme) as? GradientDrawable
    }
}